zebedee-cms/src/main/java/com/github/onsdigital/zebedee/model/Collections.java

```diff
@@ -734,7 +734,7 @@ public boolean deleteContent(
         } else if (Files.isDirectory(path)) {
             deleted = collection.deleteContentDirectory(session.getEmail(), uri);
         } else {
-            deleted = collection.deleteFile(uri);
+            deleted = deleteFile(collection, uri);
```

Comment:
Method name is slightly misleading, it's not deleting just a file, it's deleting more than that. Not sure exactly what's best but something (?) along the lines of deleteFileAndRelated()